components/AllYearView.jsx

```diff
@@ -41,13 +39,13 @@ class AllYearView extends React.Component {
         let dateMoment = moment({ year: year, month: month });
         monthsElements.push(
           <div key={month} className="EachMonth">
-            {isMonthInMetaData(this.state.allPapers, dateMoment) ? (
+            {isMonthInMetaData(allPapers, dateMoment) ? (
               <Link
-                to={getMonthPath(dateMoment)}
+                href={getMonthPath(dateMoment)}
                 title={dateMoment.format("MMMM YYYY")}
```

Comment:
this is unrelated to your changes, i think, but I get a `Warning: Failed prop type: Link: unknown props found: title, className` warning here